numpy/linalg/lapack_lite/lapack.f.patch

```diff
@@ -0,0 +1,56 @@
+@@ -57759,9 +57767,10 @@
+ *     Scan up each column tracking the last zero row seen.
+          ILACLR = 0
+          DO J = 1, N
++            I = M
+-            DO I = M, 1, -1
+-               IF( A(I, J).NE.ZERO ) EXIT
+-            END DO
++C           DO I = M, 1, -1
++C              IF( A(I, J).NE.ZERO ) EXIT
++C           END DO
```

Comment:
@eric-wieser - I was very surprised that this wouldn't just convert. Trying bit with `f2c` (as packaged by debian testing), the problem does not seem to be the `do, enddo` construction, but rather the `exit` statement.  It should not be needed to do more than replace that with a `goto <end-label>`.

p.s. I wish we could just compile the fortran; `gfortran` is a very nice compiler...